TypeCobol/Compiler/Sql/Model/SqlObject.cs

```diff
@@ -21,4 +22,18 @@ public bool AcceptVisitor(ISqlVisitor visitor)
 
         protected abstract bool VisitSqlObject([NotNull] ISqlVisitor visitor);
     }
+
+    public class TableViewCorrelationName : SqlObject
+    {
+        private SqlStorageArea _tableViewOrCorrelation;
+
+        public TableViewCorrelationName(SqlStorageArea tableViewOrCorrelation)
+        {
+            _tableViewOrCorrelation = tableViewOrCorrelation;
+        }
+        protected override bool VisitSqlObject(ISqlVisitor visitor)
+        {
+            return visitor.Visit(this);
+        }
+    }
```

Comment:
See my previous remark about this class. Also move it into its own file: add a 'Names.cs' file into '\TypeCobol\TypeCobol\Compiler\Sql\Model' and define the class in it. this file will be reused later for column name class.